classes/podsfield_pandarepeaterfield.php

```diff
@@ -612,11 +662,16 @@ public function input ( $name, $value = null, $options = null, $pod = null, $id
 							<input name="panda-repeater-start-from" id="panda-repeater-start-from-' . $ids_str . '" value="' . ( intval( $options['pandarepeaterfield_initial_amount'] ) ) . '" class="alignleft pprf-input mgr5"  type="number" step="1" min="0" autocomplete="off" title="' . __('Start from 0', 'panda-pods-repeater') . '"/>  
 							<div id="panda-repeater-load-more-button-' . $ids_str . '" class="alignleft pprf-load-more-btn mgr5" data-target="' . $ids_str . '" data-podid="' . $options['pod_id'] . '"  data-postid="' . $id . '"  data-tb="' . $savedtb_int . '" data-userid="' . $current_user->ID . '"  data-iframe_id="panda-repeater-edit-' . $ids_str . '"  data-poditemid="' . $options['id'] . '" data-cptitle="' . esc_attr( get_the_title( $options['id'] ) ) . '" data-enable-trash="' . esc_attr( $options['pandarepeaterfield_enable_trash'] ) . '"/>' . __('Load', 'panda-pods-repeater') . '</div>
 							<label class="alignleft pdt2 mgr5">' . __(' | Total items:', 'panda-pods-repeater') . ' ' . $count_int . '</label>
-							<img src = "' . PANDA_PODS_REPEATER_URL . 'images/dots-loading.gif" alt="loading" class="mgl8 alignleft pprf-ajax-img mgt13" style="display:none;"/>
+							<img src = "' . PANDA_PODS_REPEATER_URL . 'images/dots-loading.gif" alt="loading" class="mgl8 alignleft pprf-ajax-img mgt13 mgr5" style="display:none;"/>
 							<div class="pprf-load-more-report"></div>
 							
-						  </div>';	
-				}				
+						  </div> ';	
+				}	
+				if( isset( $options['pandarepeaterfield_order_by'] ) && !empty( $options['pandarepeaterfield_order_by'] )  && isset( $options['pandarepeaterfield_display_order_info'] ) && $options['pandarepeaterfield_display_order_info'] ){
+					echo '<div class="pprf-order-info pdt5 w100 alignleft" id="pprf-order-info-' . $ids_str . '">
+						  ' . $orderInfo_str . '	
```

Comment:
There's nothing in `$orderInfo_str` right now that is super concerning, but it would be a good idea to esc_html() the string as a whole.